src/components/StatusBadge/StatusBadge.test.tsx

```diff
@@ -10,15 +10,16 @@ describe("StatusBadge Test", () => {
 
     it("Shows status badge", () => {
         wrapper = Enzyme.shallow(
-            <StatusBadge level={"low"} statusBadgeText={"Registration Required"} />
+            <StatusBadge
```

Comment:
Are these `StatusBadge` changes something you've done, @misselbeck, or something showing up from a funky merge?...

Reply:
I didn't work on the `StatusBadge `component at all. It looks like these files were included as part of a merge from `development` that had a conflict with the CHANGELOG.

Reply:
Prettier formatting differences, perhaps?

Reply:
Ahh, gotit.